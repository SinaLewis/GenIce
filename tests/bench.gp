#!/usr/local/Cellar/gnuplot/5.2.7_1/bin/gnuplot

set xlabel "Number of molecules"
set ylabel "Time / msec"
set log xy
pl"bench.out" w linesp, "../../GenIce/tests/bench.out" w linesp, x
#    EOF
