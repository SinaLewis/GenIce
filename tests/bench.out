64 22.6
512 98.6
4096 938
32768 9258.6
110592 53461.4
262144 246795
